README.md

```diff
@@ -1,3 +1,23 @@
+# The Real-time Publish-Subscribe Protocol for Rust
+
+This is the implementation of the Real-time Publish-Subscribe Protocol (RTPS) DDS Interoperability Wire Protocol for Rust.
+
+The Data Distribution Service for real-time systems (DDS) is an Object Management Group (OMG) machine-to-machine connectivity framework that aims to enable scalable, real-time, dependable, high-performance and interoperable data exchanges using a publish–subscribe pattern. DDS addresses the needs of applications like air-traffic control, smart grid management, autonomous vehicles, robotics, transportation systems, power generation, medical devices, simulation and testing, aerospace and defense, and other applications that require real-time data exchange [[Wiki]][wiki-dds-url].
+
+### Features
+The objectives of this implementation are:
+* Implementing RTPS according to specification [[RTPS-2.2]][omg-rtps-url]
```

Comment:
There is also new beta version (2.3) of RTPS, when it is available, then we will migrate to the newest version